test/widget_tests/screens/screens_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:easy_study/main.dart';
+import 'package:easy_study/model/exam_type.dart';
+import 'package:easy_study/model/priority.dart';
+import 'package:easy_study/model/subject.dart';
+import 'package:easy_study/presenter/hm_map.dart';
+import 'package:easy_study/presenter/home.dart';
+import 'package:easy_study/presenter/settings.dart';
+import 'package:easy_study/presenter/subject_add.dart';
+import 'package:easy_study/presenter/subject_edit_or_delete.dart';
+import 'package:easy_study/store/app_state.dart';
+import 'package:easy_study/view/subject_card.dart';
+import 'package:easy_study/view/subject_overview.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:redux/redux.dart';
+
+void main() {
+  Subject _getDummySubject() {
+    return Subject.name("Software Engineering II", ExamType.WRITTEN_EXAM,
+        "T1.011", Priority.MINIMALISM, "A funny subject.", 5);
+  }
+
+  EasyStudyApp _createApp(Widget widget) {
+    final store =
+        new Store<AppState>(searchReducer, initialState: AppState.initial());
+    var app = new EasyStudyApp(
+      title: 'Easy Study',
+      store: store,
+    );
+    app.store.dispatch(ChangeView(widget));
+    return app;
+  }
+
+  testWidgets('show subject overview', (tester) async {
+    var createApp = _createApp(SubjectOverview());
+    await tester.pumpWidget(createApp);
+    await tester.pump();
+  });
+
+  testWidgets('show subject card', (tester) async {
```

Comment:
if you add the card test you should rebase or create an issue to add a "subjectprogressbar" test